src/action/list_collections.rs

```diff
@@ -77,11 +77,12 @@ pub struct ListCollections<'a, M = ListSpecifications, S = ImplicitSession> {
     session: S,
 }
 
-impl<'a, M> ListCollections<'a, M> {
+impl<'a, M, S> ListCollections<'a, M, S> {
```

Comment:
This is a fix for an unrelated bug I noticed while I was in there - without the type parameter, options could only be set _before_ calling `.session` (because that changes the type to `ListCollections<'a, M, ExplicitSession>`, which that impl block doesn't match).

Reply:
Ah I see. Nice find.